samples/net/vlan/prj.conf

```diff
@@ -18,6 +18,9 @@ CONFIG_NET_IF_MCAST_IPV6_ADDR_COUNT=5
 CONFIG_NET_IF_UNICAST_IPV4_ADDR_COUNT=1
 CONFIG_NET_MAX_CONTEXTS=10
 
+CONFIG_NET_IF_MAX_IPV6_COUNT=3
```

Comment:
`CONFIG_NET_IF_MAX_IPV4_COUNT=3`

Reply:
Indeed, should have been ipv4 count. Will fix.